composer.json

```diff
@@ -21,12 +21,15 @@
         "cakephp/cakephp": "^4.0",
         "cakephp/cakephp-codesniffer": "^4.0",
         "firebase/php-jwt": "^6.2",
+        "paragonie/paseto": "^2.4",
+        "paragonie/constant_time_encoding": "^2.2",
```

Comment:
This was required because of prefer-lowest on 7.2. The method needed from the package was not included until 2.2 (current is 2.5).